_private/CreateSession.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE hml>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Create Session</title>
+	<link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
+	<link href="content/style.css" rel="stylesheet">
+</head>
+
+<body>
+	<nav class="navbar navbar-inverse">
+		<div class="navbar-brand">Buzzer</div>
+	</nav>
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-10 col-xs-offset-1 col-sm-9 col-sm-offset-2">
+				<h2>Create New Session</h2>
```

Comment:
This is the title of the page, i think generally titles have capitals:
Capitalize the first word of the title, the last word of the title, and all nouns, pronouns, verbs, adverbs, adjectives, subordinating conjunctions, and a few conjunctions. Prepositions are only capitalized if they are used adjectivally or adverbially.